### PR TITLE
Add q4 roadmap

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

**Comment by curquiza** [comment]
FYI @alallema and @bidoubiwa!

Nothing should be surprising for you, but we will talk about it next Thursday in any case 🙂

File: roadmap/2021.md [hunk 1]
```diff
@@ -77,19 +77,63 @@ Misc:
...
+Hacktoberfest
+- [X] Prepare issues for Hacktoberfest
+- [ ] Handle contributors and their PRs
+
 SDKs:
 - [ ] (_Still in progress during all year long_) Improve SDKs by making more friendly methods
-- [ ] Swift SDK: normalized + error handler
 - [ ] Rust SDK: normalized + error handler
-- [ ] meilisearch-js: cache
+- [ ] JavaScript SDK: add cache
```
Comment (curquiza) [inline roadmap/2021.md:107]:
```suggestion
- [X] Instant MeiliSearch: add cache
```

**Review by tpayet — APPROVED** [review]
lgtm 🎉 Let's go 🏄🏽‍♀️

**Review by qdequele — CHANGES_REQUESTED** [review]
I see that you deprioritize the SQL connector over WordPress. Will the WordPress integration use the PHP library or will it use a direct connection to the SQL database? Or will it go through the PHP SDK implementation in a special package? Do you know technically how a SQL connector would work?

**Pull request merged**